wasm/src/programs/verifying_key/credits.rs

```diff
@@ -26,249 +26,264 @@ impl VerifyingKey {
         VerifyingKey::from_bytes(&snarkvm_parameters::testnet::BondPublicVerifier::load_bytes().unwrap()).unwrap()
     }
 
+    /// Returns the verifying key for the bond_validator function
+    ///
+    /// @returns {VerifyingKey} Verifying key for the bond_validator function
+    #[wasm_bindgen(js_name = "bondValidatorVerifier")]
+    pub fn bond_validator_verifier() -> VerifyingKey {
+        let vk = CurrentNetwork::get_credits_verifying_key("bond_validator".to_string()).unwrap().clone();
```

Comment:
I'm seeing a lot of code duplication in these methods, it's probably worthwhile to create a helper method to make the code shorter.

Reply:
I agree, just don't have the allocation to clean it up right now, I'm open to having others make the changes if you have the allocation.

Reply:
Sure, that's fine, we can do it later.